bitwarden_license/src/Scim/Groups/PostGroupCommand.cs

```diff
@@ -39,7 +39,7 @@ public async Task<Group> PostGroupAsync(Guid organizationId, ScimGroupRequestMod
         }
 
         var group = model.ToGroup(organizationId);
-        await _groupService.SaveAsync(group, EventSystemUser.SCIM, null);
+        await _createGroupCommand.CreateGroupAsync(group, EventSystemUser.SCIM, null);
```

Comment:
Are we sure that SCIM only uses POST for creating and not updating? Can you please check the spec? (and/or the spec for the specific IdPs we support)

Reply:
Apparently it can be used for updating on some endpoints (https://www.rfc-editor.org/rfc/rfc7644.html#section-3.2):
> POST   Depending on the endpoint, creates new resources, creates a
>           search request, or MAY be used to bulk-modify resources.

The only endpoints I see that use that feature are [Bulk](https://www.rfc-editor.org/rfc/rfc7644.html#section-3.7) and [Search](https://www.rfc-editor.org/rfc/rfc7644.html#section-3.4.3).

In the case of the Users and Groups endpoints they are only used for creating https://www.rfc-editor.org/rfc/rfc7644.html#section-3.3